File: eval/corpus/disambig/use_csv.py
from json_codec import json_marker
from csv_codec import csv_marker


def export_table(rows):
    """Encode a list of tabular rows as CSV text and return the spreadsheet body."""
    csv_marker()
    return encode(rows)  # ambiguous: json_codec.encode vs csv_codec.encode — CSV context

File: eval/corpus/disambig/use_csv.ts
import { jsonMarker } from "./json_codec";
import { csvMarker } from "./csv_codec";

export function exportTable(rows: Record<string, unknown>[]): string {
  // Encode a list of tabular rows as CSV text and return the spreadsheet body.
  csvMarker();
  return encode(rows); // ambiguous: json_codec.encode vs csv_codec.encode — CSV context
}
